.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@ default_language_version:
   node: latest
 exclude: '\.snap|node_modules$'
 repos:
-  - repo: git://github.com/pre-commit/pre-commit-hooks
```

Comment:
random attempt at fixing some pre-commit hooks, but they're still broken for me. I think the patch here is still absolutely necessary as `git://` is literally deprecated by github